### PR TITLE
Pull 3rd party images from our artifact registry

### PR DESCRIPTION
**Pull request by rasa-aadlv**
- Bitnami will soon stop supporting current versions
- Pull 3rd party images from our artifact registry instead of bitnami's

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -8,6 +8,9 @@ concurrency:
   group: ${{ github.head_ref }}
   cancel-in-progress: true
 
+env:
+  HELM_EXPERIMENTAL_OCI: '1'
+
```
Comment (miraai) [inline .github/workflows/ci.yml:13]: You don't need this since your Helm version (3.11.2) supports OCI.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -170,7 +173,7 @@ jobs:
       - name: Set up Helm
         uses: azure/setup-helm@v1
         with:
-          version: v3.6.0
+          version: v3.11.2
```
Comment (miraai) [inline .github/workflows/ci.yml:176]: `azure/setup-helm` should be moved to `v3` instead of `v1`.

Comment (rasa-aadlv) [reply .github/workflows/ci.yml]: Done.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (rasa-aadlv) [reply .github/workflows/ci.yml]: Done.

**Pull request merged**